apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -104,13 +124,15 @@ $tokensTable-border-width: 1px;
 
 .c-tokensTable-content {
     padding: $tokensTable-column-padding;
+    background-color: f.$color-container-default;
```

Comment:
Think this PR needs a rebase against the main branch – looks like it's got some changes that Lizzie's PR had in it?

Reply:
yep it needed, i just rebased 💯